.travis.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+ON_JAVA_8=$(echo $JAVA_HOM | grep java-8)
```

Comment:
Should be `echo $JAVA_HOME` (missing final `E`)

Reply:
You are right, done